packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -158,27 +158,35 @@ const main = async () => {
 
     managedRuntimeCommand('save-credentials')
         .description(`save API credentials for Managed Runtime`)
-        .requiredOption(
-            '-u, --user <email>',
-            'the e-mail address you used to register with Managed Runtime',
-            (val) => {
-                if (!validator.isEmail(val)) {
-                    throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
-                } else {
-                    return val
+        .addOption(
+            new program.Option(
+                '-u, --user <email>',
+                'the e-mail address you used to register with Managed Runtime',
+                (val) => {
+                    if (!validator.isEmail(val)) {
+                        throw new program.InvalidArgumentError(`"${val}" is not a valid email`)
+                    } else {
+                        return val
+                    }
                 }
-            }
+            )
+                .env('MRT_USER')
```

Comment:
yeah definitely can't completely break CLOUD_API_BASE. Theres other code that uses that for sure. But it is a weird one since nobody outside of SF would understand what "cloud api" even is. MRT_ORIGIN sounds much better

I can change the others.

Reply:
I didn't change CLOUD_API_BASE since that's internal and seems to rock the boat more for this otherwise non-breaking PR.

changed credentials file prefix to MRT but kept the old as a hidden fallback (unfortunately can't specify more than 1 env var source in the option spec)